### ISSUE TITLE
Robovac issue G10 Hybrid issue, unable to control via HA

### ISSUE THREAD

**Issue by GitHubxtsnl** [CodeFoodPixels/robovac#52]
it has been a couple of days since my  Eufy Robovac stopped working.
running HomeAssistant CORE 2024.1.6
running Eufy Robovac custom integration 1.1.1
When using the official Eufy app I can control the Robovc without any issues.
Wifi version 1.5.1
MCU version 1.3.7

errors in the log below:

```
Logger: custom_components.robovac.tuyalocalapi
Source: custom_components/robovac/tuyalocalapi.py:594
Integration: Eufy Robovac ([documentation](https://github.com/codefoodpixels/robovac), [issues](https://github.com/codefoodpixels/robovac/issues))
First occurred: 16:38:04 (212 occurrences)
Last logged: 17:48:26

'utf-8' codec can't decode bytes in position 0-1: invalid continuation byte
```
abnd this:

```
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:898
First occurred: 16:38:06 (212 occurrences)
Last logged: 17:48:28

Update for vacuum.robovac fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 898, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1216, in async_device_update
    await self.async_update()
  File "/config/custom_components/robovac/vacuum.py", line 314, in async_update
    await self.vacuum.async_get()
  File "/config/custom_components/robovac/tuyalocalapi.py", line 706, in async_get
    await self.async_update_state(response)
  File "/config/custom_components/robovac/tuyalocalapi.py", line 739, in async_update_state
    if state_message.payload and state_message.payload["dps"]:
       ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'coroutine' object has no attribute 'payload'
```

**Comment by nathbullen** [comment]
I'm running into the same issue, was working fine but have recently moved house and now am unable to set up the vacuum at my new place, hitting these same errors in my logs 🤔

**Comment by GitHubxtsnl** [comment]
I have removed the robovac from my Eufy account (iphone app) and used the Tuya Smart Life app to add the robovac to my phone. 
Unfortunately I cannot control the robovac via this Smart Life app anymore but I do can control it from HomeAssistant via the TUYA integration. Only thing which is not working is the "Return Home" function. Issue raised here:
https://github.com/home-assistant/core/issues/110968

**Comment by GitHubxtsnl** [comment]
This issue is resolved by using the latest version 1.2.8.

**Comment by GitHubxtsnl** [comment]
This issue is resolved by using the latest version 1.2.8.

**Comment by GitHubxtsnl** [comment]
This issue is resolved by using the latest version 1.2.8.
